db/seeds.rb

```diff
@@ -1,68 +1,27 @@
-s9 = Seance.create(movie_id: 9, hall_id: 2, start_time: Time.now + 3.days, price: 12)
-s10 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 1.days, price: 10)
-s11 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now, price: 10)
-s12 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 4.hours , price: 10)
-s13 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 8.hours, price: 10)
-s14 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 12.hours, price: 10)
-s15 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 16.hours, price: 10)
-s16 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 20.hours, price: 10)
-s17 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 1.days + 4.hours, price: 10)
-s18 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now + 1.days + 21.hours, price: 10)
-s19 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now - 1.days, price: 10)
-s20 = Seance.create(movie_id: 1, hall_id: 1, start_time: Time.now - 6.hours, price: 10)
-s21 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 1.hours + 30.minutes, price: 10)
-s22 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 4.hours + 50.minutes, price: 10)
-s23 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 8.hours + 20.minutes, price: 10)
-s24 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 11.hours, price: 10)
-s25 = Seance.create(movie_id: 1, hall_id: 4, start_time: Time.now + 2.days, price: 10)
-s26 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 8.days, price: 10)
-s27 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 6.days, price: 10)
-s28 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 4.days, price: 10)
-s29 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 3.days, price: 10)
-s30 = Seance.create(movie_id: 1, hall_id: 8, start_time: Time.now + 3.days + 10.hours, price: 10)
-s31 = Seance.create(movie_id: 1, hall_id: 7, start_time: Time.now + 3.days + 9.hours, price: 10)
-s32 = Seance.create(movie_id: 1, hall_id: 6, start_time: Time.now + 3.days + 11.hours, price: 10)
-s33 = Seance.create(movie_id: 1, hall_id: 5, start_time: Time.now + 9.days + 20.minutes, price: 10)
-s34 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 3.days + 20.hours, price: 10)
-s35 = Seance.create(movie_id: 1, hall_id: 4, start_time: Time.now + 5.days + 10.hours, price: 10)
-s36 = Seance.create(movie_id: 1, hall_id: 9, start_time: Time.now + 5.days + 1.hours, price: 10)
-s37 = Seance.create(movie_id: 1, hall_id: 10, start_time: Time.now + 5.days + 5.hours, price: 10)
-s38 = Seance.create(movie_id: 1, hall_id: 8, start_time: Time.now + 5.days + 7.hours, price: 10)
-s39 = Seance.create(movie_id: 1, hall_id: 7, start_time: Time.now + 5.days + 19.hours, price: 10)
+500.times do
+    Seance.create(
+        movie_id: rand(10) + 1,
```

Comment:
I think `Movie.all.sample.id` would be more elegant 😉 Same with the `hall_id`.

Reply:
Yup, looks better, thanks :).